src/java/org/apache/nutch/tools/arc/ArcSegmentCreator.java

```diff
@@ -73,18 +71,17 @@
  * </p>
  * 
  */
-public class ArcSegmentCreator extends Configured implements Tool,
-    Mapper<Text, BytesWritable, Text, NutchWritable> {
+public class ArcSegmentCreator extends Configured implements Tool {
 
   private static final Logger LOG = LoggerFactory
       .getLogger(MethodHandles.lookup().lookupClass());
   public static final String URL_VERSION = "arc.url.version";
-  private JobConf jobConf;
-  private URLFilters urlFilters;
-  private ScoringFilters scfilters;
-  private ParseUtil parseUtil;
-  private URLNormalizers normalizers;
-  private int interval;
+  private static Configuration conf;
+  private static URLFilters urlFilters;
+  private static ScoringFilters scfilters;
+  private static ParseUtil parseUtil;
+  private static URLNormalizers normalizers;
+  private static int interval;
```

Comment:
See comments in Generator etc. regarding static variables shared between mapper and reducer classes.